bin/GameIndex.yaml

```diff
@@ -130,6 +130,11 @@ PCPX-96649:
 PCPX-96660:
   name: "Tourist Trophy [Demo]"
   region: "NTSC-J"
+PCPX-98017:
+  name: "Ratchet & Clank 4th - GiriGiri Ginga no Giga Battle [Demo, Taikenba]"
```

Comment:
Typo here? Should this be Taikenba or Taikenban?

Reply:
There are several GameDB entries that are labelled as [Demo, Taikenba] but yes, it does seem to be a typo. I could fix them all if necessary.

Reply:
Don't demo and taikenban mean the same thing?  Why do we need both?

Reply:
I've decided to just label them as [Demo] since that's what they both basically mean.